pkg/distributor/distributor_test.go

```diff
@@ -276,6 +278,77 @@ func TestDistributor_Push(t *testing.T) {
 				cortex_distributor_ingester_appends_total{ingester="2",type="metadata"} 1
 			`,
 		},
+		"A push to 3 happy ingesters should succeed, histograms": {
+			numIngesters:     3,
+			happyIngesters:   3,
+			samples:          samplesIn{num: 5, startTimestampMs: 123456789000},
+			histogramSamples: true,
+			metadata:         5,
+			expectedResponse: emptyResponse,
+			metricNames:      []string{lastSeenTimestamp, distributorReceivedSamples},
+			expectedMetrics: `
+				# HELP cortex_distributor_latest_seen_sample_timestamp_seconds Unix timestamp of latest received sample per user.
```

Comment:
nit: Can we have the metrics aligned?

Reply:
Can you please elaborate on the comment? Any specific metrics to align

Reply:
Ah nvm. Are you talking about the formatting? I can take a look why it shows up like this but it looks aligned perfectly in my local

Reply:
Yeah, formatting. It should be auto corrected by pre commit checks.